popup.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    Hello, world!
+</body>
```

Comment:
We'll be adding some styling as one of the next steps to practice working with the SASS preprocessor and CSS stylesheets in general. Just a sneak peek.